lib/Webhooks.js

```diff
@@ -1,277 +1,174 @@
 'use strict';
```

Comment:
Note to self: this file to be re-reviewed after adjusting target framework.

Reply:
Same as above, resolving for now but feel free to open this back up if you want me to change something!